ocaml/typing/persistent_env.mli

```diff
@@ -73,6 +73,12 @@ val find_in_cache : 'a t -> Compilation_unit.Name.t -> 'a option
 val check : 'a t -> (Persistent_signature.t -> 'a)
   -> loc:Location.t -> Compilation_unit.Name.t -> unit
 
+(* Lets it be known that the given module is a parameter and thus is expected
+   to have been compiled as such. It may or may not be a parameter to _this_
+   module (see [register_parameter]). Raises an exception if the module has
```

Comment:
Note that `register_parameter` does not appear to exist. Is it added later? If it is then you can probably just leave this comment, but I wanted to make sure that it will exist.

Reply:
Actually, looks like I changed the name later on. Fixing the comment both here (to add "forthcoming") and in #1846 (to fix the name).